### ISSUE TITLE
[FOWT]Platform pitch and roll motions largely different from Bladed results under parked condition and specified azimuth angle.

### ISSUE THREAD

**Issue by getChaos22** [OpenFAST/openfast#2464]
Hi all,

I am currently comparing results between OpenFAST and Bladed for FOWT through the IEA 15MW wind turbine atop UMaine platform.  But I found some differences existed in the platform pitch and roll movements under some specified condition like parked with an azimuth angle of 50 degree.   

In the following picture, the results were obtained under still  water, steady wind with no wind sheer and the wind direction is -30°. The turbine was parked and the azimuth angle  is 50°. The unsteady aerodynamic or dynamic stall were both closed in Bladed and OpenFAST
![image](https://github.com/user-attachments/assets/890f43d0-23dd-440e-aba9-ed995aca03aa)

It seems like Bladed and OpenFAST show different damping effects.  

Do you have any ideas or experiences about this kind of problem? Is this related to aerodynamics or hydrodynamics? 

Thanks!
Chao Li

**Comment by jjonkman** [comment]
Dear @getChaos22.

It sounds like you are running a simplified form of DLC 6.2 (with steady wind and still water), but for a parked/idling condition with high yaw error that can result in an aeroelastic stability, leading to growth in response over time.  I agree that it appears that your OpenFAST model has more damping than your Bladed model, but I'm not sure I know why that would be (and could depend on various damping levels like structural and hydrodynamic).  DLC 6.2 is known to generate a blade-edgewise instability whereas you are showing platform roll and pitch, which is much further down the load path.  To isolate where the difference in damping levels are coming from, I would suggest to first start by comparing the blade response with a rigid support structure, and then add complexity in steps (only blade flexibility, then blade + tower flexibility, then blade + tower flexiblity + floater motion).

Best regards.

**Comment by getChaos22** [comment]
Dear @jjonkman 

Thank you very much for your quick reply. Please first apologize for my unclear description. I forgot to mention that in the cases I showed above, rigid tower and blades were employed. 
Following your advice, I have tried flexible blades with rigid tower and fixed bottom. In this situation, results of OpenFAST and Bladed do show blade instability. OpenFAST didn't finish the calculation and the error message shows that the Mach number exceeds 1.0.
![image](https://github.com/user-attachments/assets/217a3075-f39a-452b-8c75-b8fec790e59a)
![image](https://github.com/user-attachments/assets/fff9049f-3eca-4745-912a-92fe539686ab)
![image](https://github.com/user-attachments/assets/735d5109-f54a-46b3-9e63-47479f647001)

It is clear that blade responses obtained by Bladed and ENFAST are different. Basing these results and those I showed above, is it solid to say that it is aerodynamic instability causes the differences of platform motion between Bladed and OpenFAST? Even though in the floating case, the blades and tower are rigid, maybe the aeroelastic instability is reflected in platform motion? This is my guesses. If it does not make sense, please correct me. 

Thanks again.

**Comment by jjonkman** [comment]
Dear @getChaos22,

Thanks for clarifying.  

Yes, I agree that it is an aerodynamic instability that is coupling with floater motion in your original post and with blade deflection in your second post.  Again, I'm not sure why OpenFAST and Bladed do not match better for this case, but I agree that it could be due to different levels of damping.  If you disable all DOFs in both OpenFAST and Bladed, do you get similar aerodynamic loads between the models for this case (which would imply that differences in damping are more likely than differences in direct aerodynamic forcing)?

Best regards,

**Comment by getChaos22** [comment]
Dear @jjonkman 

I'm very sorry for replying so late.

Following your advice, I disabled all DOFs and checked the angle of attack (AoA) in 70% section of the 3 blades. The results are shown in the following table:
![image](https://github.com/user-attachments/assets/de3cd840-1e5b-4b3e-93ae-4de57ff21cb1)

It can be seen from the table that the AoAs of OpenFAST and Bladed are not identical. The differences of AoA of Blade 1 and Blade 2 are relatively small (smaller than 1°). The difference of the AoA of Blade 3 is relatively large. Angle of attack obtained by OpenFAST is 80% larger than Bladed. 

Do you think this difference in AoA would cause a significant difference in aerodynamic forces?

Apologize again for my late reply!

Thank you!

Best regards

**Comment by jjonkman** [comment]
Dear @getChaos22,

My understanding is that you are now modeling a rigid wind turbine with a parked rotor.  Presumably, you've disabled all induction and unsteady airfoil aerodynamics.  In this case, the angle of attack should be purely geometric, so, I don't understand why OpenFAST and Bladed would result in different angles of attack, unless the models are not set up the same way.  Can you clarify?

Best regards,

**Comment by getChaos22** [comment]
Dear @jjonkman 

Sorry for my unclear description, truly I am now modeling a rigid wind turbine with a parked rotor, and I've disabled all induction and unsteady aerodynamics. 

The models I used both come from official website or demo of IEAWindTask37 and Bladed, I didn't change anything in the airfoil information or blade geometry. The OpenFAST model is obtained from [IEA 15 MW official website](https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-UMaineSemi), and the Bladed model is from the IEA 15MW demo of Bladed 4.15. I have put the two models in the attachment file. The two models are not come from a same origin, does it mean that we cannot make sure they are identical?
[openfast_and_bladed_model.zip](https://github.com/user-attachments/files/17934142/openfast_and_bladed_model.zip)
 

Best regards,
